system/i18n/formbuilder/item-types/csrf_token.properties

```diff
@@ -0,0 +1,5 @@
+title=CSRF Token
```

Comment:
This isn't an item type really - so suggest putting that error message somewhere general like `cms.properties`.

Reply:
Found cms:invalidCsrfToken.error in the core so updating to use that instead.